tee-worker/ts-tests/integration-tests/examples/direct-invocation/util.ts

```diff
@@ -193,7 +193,7 @@ export async function createSignedTrustedCallSetIdentityNetworks(
         signer,
         mrenclave,
         nonce,
-        [subject, subject, identity, web3networks, hash]
+        [subject.toHuman(), subject.toHuman(), identity, web3networks, hash]
```

Comment:
Maybe someone can explain to me why we need `.toHuman()` here ?

Reply:
Because `createType` is picky when [handling enum-typed inputs](https://github.com/polkadot-js/docs/issues/223).